### PR TITLE
Update Ansible version to 2.16.6 for Community EE Base and Minimal releases

### PR DESCRIPTION
**Pull request by anweshadas**

### CODE REVIEW

File: .github/workflows/eerelease.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+name: Release Excution Environments
+on:
+
```
Comment (felixfontein) [inline .github/workflows/eerelease.yml:3]: I think this should not be part of this PR.

Comment (anweshadas) [reply .github/workflows/eerelease.yml]: Yes,  I pushed it by mistake. Removed it now.

**Review by felixfontein — APPROVED** [review]
